packages/sx05re/emuelec/config/emuelec/scripts/show_splash.sh

```diff
@@ -14,42 +14,53 @@ DEFAULTSPLASH="/storage/.config/splash/splash-1080.png"
 VIDEOSPLASH="/usr/config/splash/emuelec_intro_1080p.mp4"
 DURATION="5"
 
+# we make sure the platform is all lowercase
+PLATFORM=${PLATFORM,,}
+
 case $PLATFORM in
- "ARCADE"|"FBA"|"NEOGEO"|"MAME"|CPS*)
-   PLATFORM="ARCADE"
+ "arcade"|"fba"|"neogeo"|"mame"|cps*)
+   PLATFORM="arcade"
   ;;
- "RETROPIE")
+ "retropie")
    # fbterm does not like the splash screen 
    exit 0
   ;;
 esac
 
 if [ "$PLATFORM" == "intro" ] || [ "$PLATFORM" == "exit" ]; then
 	SPLASH=${DEFAULTSPLASH}
-elif [ "$PLATFORM" == "default" ]; then
-	SPLASH=${GAMELOADINGSPLASH}
+#elif [ "$PLATFORM" == "default" ]; then
+#	SPLASH=${GAMELOADINGSPLASH}
 else
 	SPLASHDIR="/storage/overlays/splash"
 	ROMNAME=$(basename "${2%.*}")
 	SPLMAP="/emuelec/bezels/arcademap.cfg"
 	SPLNAME=$(sed -n "/`echo ""$PLATFORM"_"${ROMNAME}" = "`/p" "$SPLMAP")
 	REALSPL="${SPLNAME#*\"}"
 	REALSPL="${REALSPL%\"*}"
-	if [[ -d "$SPLASHDIR/$PLATFORM" ]]; then
-		[ ! -z "$REALSPL" ] && SPLASH1=$(find $SPLASHDIR/$PLATFORM -iname "$REALSPL*.mp4" -maxdepth 1 | sort -V | head -n 1)
-		[ -z "$SPLASH1" ] && SPLASH1=$(find $SPLASHDIR/$PLATFORM -iname "$REALSPL*.png" -maxdepth 1 | sort -V | head -n 1)
-		[ ! -z "$ROMNAME" ] && SPLASH2=$(find $SPLASHDIR/$PLATFORM -iname "$ROMNAME*.mp4" -maxdepth 1 | sort -V | head -n 1)
-		[ -z "$SPLASH2" ] && SPLASH2=$(find $SPLASHDIR/$PLATFORM -iname "$ROMNAME*.png" -maxdepth 1 | sort -V | head -n 1)
-		
-		SPLASH3="$SPLASHDIR/$PLATFORM/splash.mp4"
-		[ ! -f "$SPLASH3" ] && SPLASH3="$SPLASHDIR/$PLATFORM/splash.png"
-	fi	
+[ ! -z "$REALSPL" ] && SPLASH1=$(find $SPLASHDIR/$PLATFORM -iname "$REALSPL*.png" -maxdepth 1 | sort -V | head -n 1)
+[ ! -z "$ROMNAME" ] && SPLASH2=$(find $SPLASHDIR/$PLATFORM -iname "$ROMNAME*.png" -maxdepth 1 | sort -V | head -n 1)
+[ ! -z "$REALSPL" ] && SPLASHVID1=$(find $SPLASHDIR/$PLATFORM -iname "$REALSPL*.mp4" -maxdepth 1 | sort -V | head -n 1)
+[ ! -z "$ROMNAME" ] && SPLASHVID2=$(find $SPLASHDIR/$PLATFORM -iname "$ROMNAME*.mp4" -maxdepth 1 | sort -V | head -n 1)
+
+SPLASH3="$SPLASHDIR/$PLATFORM/launching.png"
+SPLASHVID3="$SPLASHDIR/$PLATFORM/launching.mp4"
+	
 	if [ -f "$SPLASH1" ]; then
 		SPLASH=$SPLASH1
+		if [ -f "$SPLASHVID1" ]; then
```

Comment:
The videosplash will ONLY show if SPLASH1 exists, is this intended? if so, why? it should also work if only SPLASHVID1 exists.

Reply:
I prefer video splash rather then picture splash. 
For example, if we have dino.zip for arcade, and we have dino.mp4 and dino.png as splash at same time, dino.mp4 will be showed. 
If only dino.png is present, then it will be showed as splash. 
If both dino.mp4 and dino.png are absent, then we will look for arcade/launching.mp4 or arcade/launching.png. If they are not there, we will use ${GAMELOADINGSPLASH}.

Reply:
Yeah but my comment was if i have "dino.mp4" present and "dino.png" is abset then the default will be shown instead of "dino.mp4" that means you need both, dino.png and dino.mp4 this does not seem right
I think a better way would be 

```
if [ -f "$SPLASHVID1" ]; then
        SPLASH=$SPLASHVID1
    elif [ -f "$SPLASH1" ]; then
        SPLASH=$SPLASH1
    elif [ -f "$SPLASHVID2" ]; then
        SPLASH=$SPLASHVID2
    elif [ -f "$SPLASH2" ]; then
        SPLASH=$SPLASH2
    elif [ -f "$SPLASHVID3" ]; then
        SPLASH=$SPLASHVID3
    elif [ -f "$SPLASH3" ]; then
        SPLASH=$SPLASH3
    else
        SPLASH=${GAMELOADINGSPLASH}
    fi
```
Priority would be:

video splash1 (romname)
image splash 1 (romname)
video splash 2 (system)
image splash 2 (system)
video splash 3 (launching.mp4)
image splash3 (launching.png)

Reply:
Yes, you are right. I missed that.